Content/0.4.13 - Security Factors/securityFactors.json

```diff
@@ -1,6 +1,8 @@
 {
 	"securityFactors_title": "Security Factors",
-	"securityFactors_subtitle": "View and manage your security factors",
+	"securityFactors_subtitle": "Manage the security factors you’ll use in your Security Shields.",
+	"securityFactors_hardware": "Hardware",
+	"securityFactors_information": "Things you know",
```

Comment:
after these changes have been integrated in both platforms, we can delete every other string besides these 4